config/Dockerfiles/cloud-image-compose/virt-customize.sh

```diff
@@ -74,20 +74,20 @@ enabled=1
 gpgcheck=0
 EOF
 
-koji_repo=${branch}
 if [ "${branch}" != "rawhide" ]; then
-    koji_repo="f${branch}-build"
     if ! virt-customize --selinux-relabel --memsize 4096 -a ${DOWNLOADED_IMAGE_LOCATION} --run-command "dnf config-manager --set-enable updates-testing updates-testing-debuginfo" ; then
         echo "failure enabling updates-testing repo"
         exit 1
     fi
 else
     virt-customize -a ${DOWNLOADED_IMAGE_LOCATION} --run-command "source /etc/os-release; rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-\$VERSION_ID-fedora"
 fi
+
+koji_repo=$(echo ${DIST_BRANCH}-build | sed -e s'/fc/f/')
```

Comment:
how does `$DIST_BRANCH` look on rawhide? is that `f30` now? Or is it `master`?

Reply:
It should be `fc30`.

Reply:
In that case, let's merge it and give it a try!

Reply:
Yep, $DIST_BRANCH should *always* be fcXX where XX is the major release rawhide maps to